### PR TITLE
feat: update stars and forks stats on repo pages

### PR DESCRIPTION
**Pull request by zeucapua**
## Description

Replaces stars and forks `MetricCard`s with `RepositoryStatCard`s that show total (all-time), as well as total and average for the given range.

## Related Tickets & Documents
Relates to #3165 

## Mobile & Desktop Screenshots/Recordings
image

## Tier (staff will fill in)

- [ ] Tier 1
- [x] Tier 2
- [ ] Tier 3
- [ ] Tier 4

## [optional] What gif best describes this PR or how it makes you feel?

crunch-the-numbers-crunch-gif

### CODE REVIEW

**Comment by brandonroberts** [comment]
Looks good! Mobile is slightly off on Over 30 days avg

**Comment by nickytonline** [comment]
> Looks good! Mobile is slightly off on Over 30 days avg
> 
> 

We can change the font size on mobile to be a bit smaller, which should help with this.

File: components/Workspaces/RepositoryStatCard.tsx [hunk 1]
```diff
@@ -123,8 +143,14 @@ export const RepositoryStatCard = ({ stats, type, isLoading, hasError }: Reposit
...
                 return (
                   <tr key={stat} className="flex flex-col">
-                    <th scope="row" className="capitalize font-normal text-lg lg:text-sm text-light-slate-12 text-left">
-                      {stat.replace("_", " ")}
+                    <th scope="row" className="capitalize font-normal text-xs lg:text-sm text-light-slate-12 text-left">
+                      {type === "stars" || type === "forks"
+                        ? stat === "over_range"
+                          ? `Over ${stats.range} Days`
+                          : stat === "average_over_range"
+                          ? `Avg. Over ${stats.range} Days`
```
Comment (bdougie) [inline components/Workspaces/RepositoryStatCard.tsx:151]:
```suggestion
                          ? `Avg. per day`
```

This is confusing "Avg. over 30 days." It'd make more send to stat "Average per day."

**Review by brandonroberts — APPROVED** [review]

**Review by nickytonline — APPROVED** [review]
Took it for a test spin and works as expected and looks good on smaller screens as well.

CleanShot 2024-04-25 at 09 58 53

:shipit:

**Pull request merged**